### DISCUSSION TITLE
Windows keys CTRL-A+V+... do not work as expected in config-text-boxes

### DISCUSSION THREAD

**Discussion by gitgetgot** [ghDaYuYu/foo_discogs#62] [Spam/Abuse]
Version: 1.0.4
Compiled: Aug 19 2021

Hello,

when I put the cursor in the red marked textboxes, I cannot use the normal windows keys for select all (CTRL+A), copy (CTRL+C) or paste (CTRL+V), nor can I use the enter (ENTER) key to add some additional lines.
However, the keys are working in single line textflields (green marked)

![notyes](https://user-images.githubusercontent.com/974980/130284410-5c9b9f20-7951-4b6a-81b0-4721ab89ef95.png)
![not](https://user-images.githubusercontent.com/974980/130284416-119a53d9-1ab5-4b4c-8f1f-e624437f3ac8.png)

Is this a bug, or a feature? :-)

**Comment by ghuDaYuYu** [comment]
Just tested copy/paste is working here. Have you tried with a single word?
The text in those textboxes are a representation of a single line formatting string.
If you copy formatting string to notepad you will appreciate it real format.
No carriage returns allowed.

**Reply by gitgetgot** [reply]
>  No carriage returns allowed.
Ok.

Maybe its the OS? Win7 is used here.

When I copy just a letter, say X. from notepad++ or from inside the properties window of a track in fb2k, I cannot paste either.

(I think, I remember having this problem too with a preferences dialogue in ColumnsUI some months ago, and it was fixed by musicmusic, but I can't find the posts atm...)

I can use the right-click context menus, that woud do the job for now,
thanks!
